sources/tools/Stride.TextureConverter/Frontend/TexAtlas.cs

```diff
@@ -45,7 +45,6 @@ public TexAtlas(TexLayout layout, TexImage atlas)
             CurrentLibrary = atlas.CurrentLibrary;
             LibraryData = atlas.LibraryData;
             Layout = layout;
-            Name = "";
```

Comment:
Most properties are copied, even reference types, it seemed to make more sense to copy the name as well.